game/rooms/icematch.js

```diff
@@ -0,0 +1,64 @@
+const { Room } = require('colyseus')
+const GameState = require('../state/icematch')
+
+class IceRoom extends Room {
+  onInit () {
+    this.setState(
+      new GameState()
+    )
+    this.playerMoveSet = {}
+  }
+
+  onJoin (client) {
+    this.state.onJoin(client)
+
+    this.playerMoveSet[client.sessionId] = {
+      up: false,
+      down: false,
+      left: false,
+      right: false
+    }
+
+    console.log(`Player ${client.sessionId} joined!`)
+  }
+
+  onMessage (client, data) {
+    const direction = Object.keys(data)[0]
+    this.state.updatePlayerMoveSet(client, direction, data[direction])
```

Comment:
maybe `updatePlayerDirection`? `MoveSet` makes me think that you're changing the number of valid directions